pkg/controller_helper/modelsource/modelhub.go

```diff
@@ -157,3 +160,7 @@ func (p *ModelHubProvider) InjectModelLoader(template *corev1.PodTemplateSpec, i
 		},
 	})
 }
+
+func spreadEnvToInitContainer(containerEnv []corev1.EnvVar, initContainer *corev1.Container) {
```

Comment:
Can we add a test to make sure only the envs we set in the Playground or BackendRuntime will be spread to the initContainers?

Reply:
Added. PTAL. 
/kind feature